pkgs/by-name/al/alive2/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  clangStdenv,
+  fetchFromGitHub,
+  re2c,
+  z3,
+  hiredis,
+  llvm_18,
+  cmake,
+  ninja,
+  git,
+}:
+
+clangStdenv.mkDerivation (finalAttrs: {
+  pname = "alive2";
+  version = "unstable-2024-09-23";
+
+  src = fetchFromGitHub {
+    owner = "AliveToolkit";
+    repo = "alive2";
+    rev = "05a964284056b38a6dc1f807e7acad64a0308328";
+    sha256 = "sha256-9Wwt5p6Zg+A8CRs5DSMqxMmapNRBzdL5IzaqXp0drDA=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
```

Comment:
Is git still needed even with `postPatch` replacing git command?

Reply:
yes, otherwise it errors out with : 

```
       > CMake Error at /nix/store/ih3wsahlr3d787jc4kzqizp6syq6hy29-cmake-3.29.3/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
       >   Could NOT find Git (missing: GIT_EXECUTABLE)
       > Call Stack (most recent call first):
       >   /nix/store/ih3wsahlr3d787jc4kzqizp6syq6hy29-cmake-3.29.3/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
       >   /nix/store/ih3wsahlr3d787jc4kzqizp6syq6hy29-cmake-3.29.3/share/cmake-3.29/Modules/FindGit.cmake:128 (find_package_handle_standard_args)
       >   CMakeLists.txt:50 (find_package)
```

Reply:
Try to also add `--replace-fail 'find_package(Git REQUIRED)' ""`.

> Then remove git.

Please give useful advice.

Reply:
Added this `--replace-fail 'find_package(Git REQUIRED)' ""` and there was no longer a need for git. Thanks.